tasks/managed/update-trusted-tasks/update-trusted-tasks.yaml

```diff
@@ -25,6 +25,12 @@ spec:
   steps:
     - name: update-trusted-tasks
       image: quay.io/konflux-ci/appstudio-utils@sha256:591af845d7c700a178b3738e9725880e79cf63521a906325185bfe74d2a28407
+      computeResources:
```

Comment:
@avi-biton do these compute resource limits sound good to you? I wasn't able to get any metrics on previous runs for this task

Reply:
It sound good to me, I do not think we need more resources.
And in case we will we can always update it :)

Reply:
thanks!